package-builder/flox-build.mk

```diff
@@ -273,7 +273,7 @@ define BUILD_local_template =
 	$(_VV_) $(_rm) -rf $(_out)
 	$(_V_) \
 	  $(if $(_virtualSandbox),$(PRELOAD_VARS) FLOX_SRC_DIR=$$$$($(_pwd)) FLOX_VIRTUAL_SANDBOX=$(_sandbox)) \
-	  $(FLOX_INTERPRETER)/activate --env $(FLOX_ENV) --mode dev --turbo -- \
+	  $(FLOX_INTERPRETER)/activate --env $(FLOX_ENV) --mode dev --turbo --env-project $(CURDIR) -- \
```

Comment:
blocking question: do we need to add this for any of the activate invocations in build-manifest.nix?

Reply:
I think so? I didn't even know this file existed. As far as I can tell, we need `$FLOX_ENV_PROJECT` set for anything other than `mkContainer`.